### ISSUE TITLE
Add Rythm to Benchmark

### ISSUE THREAD

**Issue by jkratz55** [jreijn/spring-comparing-template-engines#16]
I'd be interested to see how Rythm stacks up since it is strongly typed and get compiled to byte code. The website claims it is faster than velocity.

**Comment by jreijn** [comment]
Rythem was already in the project, but it broke after the upgrade to Spring 4. There is currently an outstanding issue for this: https://github.com/greenlaw110/spring-rythm/issues/9

**Comment by Vest** [comment]
I have updated the mentioned issue. If I do not get the response in few days, I will prepare the PR to remove Rythm. Life is too short to wait for fixes :)

**Comment by adalbert-h** [comment]
I just came across this project and found a very fast hotfix based on mentioned issue 9 from rythmengine.

You can create simple class like this:
```java
package com.jeroenreijn.examples.rythm;

import org.rythmengine.spring.web.RythmConfigurer;
import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;

public class Configurer extends RythmConfigurer {

    @Override
    public void configurePathMatch(PathMatchConfigurer configurer) {

    }

}
```

And update mvc-dispatcher-servlet.xml:
```xml
<!-- Rythm -->
<!-- does not seem to work with Spring4.
     Known issue: https://github.com/greenlaw110/spring-rythm/issues/9  -->
<bean id="rythmConfigurer" class="com.jeroenreijn.examples.rythm.Configurer"> // Here was "org.rythmengine.spring.web.RythmConfigurer"
  <property name="resourceLoaderPath" value="/WEB-INF/rythm/"/>
  <property name="autoImports"
            value="com.jeroenreijn.examples.model.*"/>
</bean>
<bean class="org.rythmengine.spring.web.RythmViewResolver">
  <property name="order" value="0"/>
  <property name="cache" value="true"/>
</bean>
```

With this simple change it is working for me.

**Comment by Vest** [comment]
I know this fix. I have found it in Internet. What is interesting, people also include the property related to the suffix handling. I do no remember details, that is why I would avoid fixing the third party library. 

My feeling is that the owner doesn't have resources to support it, that is why I'd better remove it from the project.

**Comment by Vest** [comment]
The issue is resolved by PR #49 
I don't remember what was in the past, but with the new project structure I was able to make Rythm working.
The results in README.md are updated with the new values that I measured.
